schemas/generateBarcode.json

```diff
@@ -0,0 +1,51 @@
+{
+  "definitions": {
+    "createCashDepositRelationships": {
+      "type": "object",
+      "properties": {
+        "account": {
+          "$ref": "types.json#/definitions/Relationship"
+        },
+        "customer": {
+          "$ref": "types.json#/definitions/customersRelationship"
+        }
+      },
+      "required": [
+        "account",
+        "customer"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "title": "Cash Deposit Barcode",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": [
+        "cashDepositBarcode"
+      ]
```

Comment:
I would add a ```default``` attribute with "cashDepositBarcode" to make it easier to create the request object